listener.go

```diff
@@ -12,5 +12,6 @@ type Listener interface {
 	Directive(unused bool, qualifier, mechanism, value, effectiveValue string)
 	NonMatch(qualifier, mechanism, value string, result Result, err error)
 	Match(qualifier, mechanism, value string, result Result, explanation string, ttl time.Duration, err error)
+	FireFirstMatch(r Result, err error)
```

Comment:
```suggestion
	FirstMatch(r Result, err error)
```
It is a __Listener__ pattern, so methods are merely callbacks; `fire` is being used for `sending` events

Reply:
done